electricpy/visu.py

```diff
@@ -9,7 +9,10 @@
 
 import numpy as _np
 import matplotlib.pyplot as _plt
-
+import cmath
+import Geometry
```

Comment:
Taking other comments into consideration, these imports should be refactored to resemble:
```python
from electricpy import geometry
from electricpy.geometry import Point
from electricpy.geometry.circle import Circle
```

Note that the capitalization of `Point` and `Circle` (classes) is "pythonic" since classes should start with an uppercase letter, or in other words, be "PascalCase"

Reply:
done !!!!!